core/chapters/c06_lists.py

```diff
@@ -420,95 +420,277 @@ class index_error(Step):
         def check(self):
             return "IndexError" in self.result
 
-    class introducing_len_and_range(VerbatimStep):
+    class improve_with_list_and_loop(VerbatimStep):
```

Comment:
I'd prefer to leave the names the same in earlier steps, so if someone was about to do `introducing_len_and_range` before then it won't skip content for them.

Reply:
Hmm... OK

Reply:
specifically leave the names of the first 4 steps the same